apps/webstore/src/app/shared/enteties/design.ts

```diff
@@ -0,0 +1,9 @@
+export class TreeDesign {
```

Comment:
This class does not seem to be utilized by anything
If its purpose is to specify the data the tree design includes, and would be used in the future, it should be an interface or model within a library, not directly tied to the webstore app